### PR TITLE
Add .terraform in .dockerignore files

### LINKED ISSUE #1866: Every bundle needs a .dockerignore file that excludes the .terraform directory
When working with TF locally a .terraform directory gets created. If this is copied into the Porter bundle then porter bundle will not build as looks for az cli config.

Each bundle needs a .dockerignore such as https://github.com/marrobi/AzureTRE/blob/710b2155db221bf143a57e972d2cea26db663425/templates/workspace_services/guacamole/.dockerignore . At present the file exists in some, but not others.

### PR DESCRIPTION
**Pull request by Sonali-Rajput**
resolves #1866

## What is being addressed

When working with TF locally a .terraform directory gets created. If this is copied into the Porter bundle then porter bundle will not build as looks for az cli config.

## How is this addressed

- Add .terraform in .dockerignore .

### CODE REVIEW

**Comment by marrobi** [comment]
Thank you for your contribution, the file isn't needed in the devops directory as it is not used when building docker images.

It is each folder in the /templates/ directory that has a /terraform/ sub directory that needs it.

**Review by marrobi — CHANGES_REQUESTED** [review]
Thank you for your contribution, the file isn't needed in the devops directory as it is not used when building docker images.

It is each folder in the /templates/ directory that has a /terraform/ sub directory that needs it.

**Comment by Sonali-Rajput** [comment]
> Thank you for your contribution, the file isn't needed in the devops directory as it is not used when building docker images.
> 
> It is each folder in the /templates/ directory that has a /terraform/ sub directory okay I ll update my pr accordingly

> Thank you for your contribution, the file isn't needed in the devops directory as it is not used when building docker images.
> 
> It is each folder in the /templates/ directory that has a /terraform/ sub directory that needs it.

 okay thanks for informing this I'll update my pr accordingly😊

**Comment by Sonali-Rajput** [comment]
@marrobi can you please review my changes 🥺

**Comment by marrobi** [comment]
The other bundles missing a .dockerignore need one adding.

For example  the folders in `templates/workspace_services/guacamole/user_resources`

As per the issue, any under `./templates/`, with a `porter.yaml` and `./terraform` directory also need a `.dockerignore` file excluding terraform.

File: templates/workspace_services/gitea/.dockerignore [hunk 1]
```diff
@@ -4,3 +4,6 @@
 Dockerfile.tmpl
 terraform/deploy.sh
 terraform/destroy.sh
+
+# Local .terraform directories
+**/.terraform/*
```
Comment (martinpeck) [inline templates/workspace_services/gitea/.dockerignore:9]: small change: can you please add a new line to the end of the file?

**Review by martinpeck — CHANGES_REQUESTED** [review]
Thanks for this. Can you please just add a small change (new line to the end of the file)?

**Comment by Sonali-Rajput** [comment]
> Thanks for this. Can you please just add a small change (new line to the end of the file)?

can You please review?

**Pull request merged**